src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/reporting/ReportingImporter.kt

```diff
@@ -24,6 +24,7 @@ class ReportingImporter {
                 moveFiles = movesContent,
                 journeyFiles = journeysContent,
                 eventFiles = eventsContent,
+//                eventFiles = listOf(),
```

Comment:
Remove this if redundant.

Reply:
Having to cater for staging not having event reports in the correct (v2) format